airflow/www/templates/airflow/dags.html

```diff
@@ -401,15 +401,15 @@ <h2>DAGs</h2>
         .attr('transform', function(d, i) {
           x = (i * (diameter + circle_margin)) + (diameter/2 + circle_margin);
           y = (diameter/2) + stroke_width_hover;
-          return 'translate(' + x + ',' + y + ')';
+          return `translate(${x},${y})`;
         });
 
       g.append('svg:a')
         .attr('href', function(d) {
-          return '{{ url_for('DagRunModelView.list') }}?_flt_3_dag_id=' + dag_id + '&_flt_3_state=' + d.state;
+          return `{{ url_for('DagRunModelView.list') }}?_flt_3_dag_id=${dag_id}&_flt_3_state=${d.state}`;
         })
       .append('circle')
-        .attr('id', function(d) {return 'run-' + dag_id.replace(/\./g, '_') + d.state || 'none'})
```

Comment:
This is the bug I'm referring to. 

Sample code:
```javascript
'a-' + null || 'none'
```

This gives us `a-null`, while we actually expect `a-none`. So this should be
```javascript
'a-' + (null || 'none')
```

But of course this is fixed if we use template string for this.